login-workflow/src/new-architecture/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.test.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import '@testing-library/jest-dom';
 import { cleanup, render, screen, fireEvent } from '@testing-library/react';
 import { ForgotPasswordScreenBase } from './ForgotPasswordScreenBase';
+import { defaultProps as authContextProps } from '../../contexts/AuthContext/AuthContextProvider.test';
+import { AuthContextProvider } from '../../contexts';
 
```

Comment:
When we are fixing a bug like this, we should be adding a test case that covers the condition that was missed previously (i.e. we should have a test case that tests the default value to make sure it works, and we should have another test case that checks that a custom value for the success screen prop works).